#!/bin/sh -e
##
## 96_debian_alsaconf_improve_intro.dpatch by Thomas Hood
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Improve introductory message

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad alsa-utils-1.0.8/alsaconf/alsaconf.in /tmp/dpep.BkK00X/alsa-utils-1.0.8/alsaconf/alsaconf.in
--- alsa-utils-1.0.8/alsaconf/alsaconf.in	2005-02-18 21:41:36.000000000 +0100
+++ /tmp/dpep.BkK00X/alsa-utils-1.0.8/alsaconf/alsaconf.in	2005-02-18 21:41:37.000000000 +0100
@@ -341,9 +341,7 @@
     Advanced Linux Sound Architecture (ALSA) driver.
 
 
-  If ALSA is already running, you should close all sound
-  apps now and stop the sound driver.
-  alsaconf will try to do this, but it's not 100%% sure." $version)
+  You should stop all sound applications now." $version)
   $DIALOG --msgbox "$msg" 20 63 || acex 0
 }
 
